[install-kubeadm]: Update instructions to disable or tolerate swap (#47710)
kubernetes/website: content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

* Full network connectivity between all machines in the cluster (public or private network is fine).
* Unique hostname, MAC address, and product_uuid for every node. See [here](#verify-mac-address) for more details.
* Certain ports are open on your machines. See [here](#check-required-ports) for more details.

> * Swap configuration: The default behavior of a kubelet is to fail to start if swap memory is detected on a node.

Reviewer: We're describing prerequisites here. Turning off swap used to be a prerequisite, but actually it no longer is. We should move the whole discussion later in the page.

Author: Yeah that makes sense.
WDYT about between `Check required ports` and `Installing a container runtime`?

  This means that swap should either be disabled or tolerated by kubelet.

>   * To tolerate swap, add `failSwapOn=false` to kubelet configuration or as a command line argument.

Reviewer 2: ```suggestion
  * To tolerate swap, add `failSwapOn: false` to kubelet configuration or as a command line argument.
```

>     Note: even if `failSwapOn=false` is provided, workloads wouldn't have swap access by default.

Reviewer 2: ```suggestion
    Note: even if `failSwapOn: false` is provided, workloads wouldn't have swap access by default.
```

>     This can be changed by setting a `swapBehavior` that's different than `NoSwap` (which is the default).

Reviewer: I would make it clear where `swapBehavior` goes (people might think it goes inside a Pod spec for example).
```suggestion
    This can be changed by setting a `swapBehavior`, again in the kubelet configuration file. To use swap,
   set a `swapBehavior` other than the default `NoSwap` setting.
```
